thesis-de/content-de.tex

```diff
@@ -2670,6 +2670,45 @@ \section{Realisierung Grafana}
 
 
 \section{Tests und Qualitätssicherung}
+Für die Tests und Qualitätsicherung wurden sogenannte Unit Tests verwendet.
+Unit Test sind Software Tests und werden immer dann verwendet, wenn jeweils die
+Kernfunktion von einem Quellcode geprüft werden soll. Das Framework hierzu wird
+von rspec zur Verfügung gestellt und wurde für die im Projekt verwendeten
+Module angepasst.  Man kann hier mit einem Test, zum Beispiel prüfen, ob das
+Profil auf unterschiedlichen Betriebssystemen ausgeführt werden kann oder ob
+bestimmte Ressourcen in dieser Datei enthalten sind. Zu Beginn müssen diese
+vorab mit rspec in dem katalog kompliert und definiert werden, damit diese
+verwendet werden können. Mit folgendem Befehl, können die Tests durchgeführt
+werden:
+\begin{lstlisting}[language=bash]
+bundle exec rake spec
+\end{lstlisting}
+
+Im Beispiel im Grafana Profil, sehen die Tests wie folgt aus:
+
+\lstinputlisting[language=ruby,caption={test für grafana profil}]{../listings/rspec-grafana.txt}
+
+Zuerst, werden die verschiedenen Betriebssysteme für das Grafana Profil
```

Comment:
ich hab in meinem abschnitt extra erklärt was facts sind, kannst dich dann drauf beziehen.